src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/DruidCombatAI.java

```diff
@@ -0,0 +1,340 @@
+import com.minecolonies.coremod.entity.pathfinding.MinecoloniesAdvancedPathNavigate;
+import com.minecolonies.coremod.entity.pathfinding.pathjobs.AbstractPathJob;
+import com.minecolonies.coremod.entity.pathfinding.pathjobs.PathJobCanSee;
+import com.minecolonies.coremod.entity.pathfinding.pathjobs.PathJobMoveAwayFromLocation;
+import com.minecolonies.coremod.entity.pathfinding.pathjobs.PathJobMoveToLocation;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.ai.attributes.Attributes;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.item.PotionItem;
+import net.minecraft.potion.*;
+import net.minecraft.util.Hand;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.function.BiPredicate;
+
+import static com.minecolonies.api.research.util.ResearchConstants.DRUID_USE_POTIONS;
+import static com.minecolonies.api.util.constant.GuardConstants.*;
+import static com.minecolonies.coremod.entity.ai.basic.AbstractEntityAIFight.SPEED_LEVEL_BONUS;
+
+/**
+ * Druid combat AI
+ */
+public class DruidCombatAI extends AttackMoveAI<EntityCitizen>
+{
+    /**
+     * List of potential positive effects.
+     */
+    private static final ImmutableList<Effect> SUPPORT_EFFECTS = ImmutableList.of(Effects.DAMAGE_BOOST, Effects.SATURATION, Effects.HEAL, Effects.DAMAGE_RESISTANCE);
+
+    /**
+     * List of potential positive effects.
+     */
+    private static final ImmutableList<Effect> ADVERSE_EFFECTS = ImmutableList.of(Effects.MOVEMENT_SLOWDOWN, Effects.WEAKNESS);
+
+    /**
+     * The xp per thrown potion
+     */
+    private static final double PER_POTION_XP = 0.05D;
+
+    /**
+     * The value of the speed which the guard will move.
+     */
+    private static final double COMBAT_SPEED = 1.0;
+
+    /**
+     * Potion velocity.
+     */
+    public static final float POTION_VELOCITY = 0.5f;
+
+    /**
+     * Flee chance
+     */
+    private static final int FLEE_CHANCE = 3;
+
+    /**
+     * The parent combat AI.
+     */
+    private final AbstractEntityAIGuard<JobDruid, AbstractBuildingGuards> parentAI;
+
+    /**
+     * The combat pathing options.
+     */
+    private final PathingOptions    combatPathingOptions;
+
+    /**
+     * If the last attack was an instant effect potion.
+     */
+    private boolean instantEffect;
+
+    public DruidCombatAI(
+      final EntityCitizen owner,
+      final ITickRateStateMachine stateMachine,
+      final AbstractEntityAIGuard parentAI)
+    {
+        super(owner, stateMachine);
+
+        this.parentAI = parentAI;
+        combatPathingOptions = new PathingOptions();
+        combatPathingOptions.setEnterDoors(true);
+        combatPathingOptions.setCanOpenDoors(true);
+        combatPathingOptions.setCanSwim(true);
+        combatPathingOptions.withOnPathCost(0.8);
+        combatPathingOptions.withJumpCost(0.01);
+        combatPathingOptions.withDropCost(0.9);
```

Comment:
hm that one may be not ideal for a ranged type, it'll prefer dropping over a normal path. Probably wrong for rangers aswell. Should likely be rather high compared to normal to have them prefer staying on higher ground

Reply:
Set it to 1.5 for both (default is 1.3)